### PR TITLE
SOOP ASF MT - handle ISAR QC files in pipeline

### PR DESCRIPTION
**Pull request by lbesnard**

### CODE REVIEW

**Comment by lbesnard** [comment]
linked to https://github.com/aodn/harvesters/pull/525

@lwgordonimos  FYI
watch.d script modified, so will need to be restarted

File: watch.d/SOOP_ASF_MT.json [hunk 1]
```diff
@@ -4,5 +4,5 @@
       "SOOP/ASF/MT"
   ],
   "execute": "bin/generic_incoming_handler.sh",
-  "execute_params": "--exec SOOP/SOOP_ASF_SST_XBT/soop_asf_sst_pipeline/dest_path.py --regex '^IMOS_SOOP-ASF_[A-Z]*_[0-9]{8}T[0-9]{6}Z_.*_FV0[0-1]_C-[0-9]{8}T[0-9]{6}Z.nc$' --email laurent.besnard@utas.edu.au"
+  "execute_params": "--exec SOOP/SOOP_ASF_SST_XBT/soop_asf_sst_pipeline/dest_path.py --regex '^IMOS_SOOP-ASF_[A-Z]*_[0-9]{8}T[0-9]{6}Z_.*_FV0[0-1].*_C-[0-9]{8}T[0-9]{6}Z.nc$' --email laurent.besnard@utas.edu.au"
```
Comment (mhidas) [inline watch.d/SOOP_ASF_MT.json]: The regex could be even a bit more strict matching the file version field `_FV0[0-1]_.*`

Comment (lbesnard) [inline watch.d/SOOP_ASF_MT.json]: this kind of shift the eventual problem to the next part '_C'

I added another commit which should handle both cases

**Comment by mhidas** [comment]
:+1: Ok, do you want to squash it?

**Comment by lbesnard** [comment]
@just did it, thanks

**Pull request merged**